src/library/scala/collection/Iterable.scala

```diff
@@ -431,16 +431,7 @@ trait IterableOps[+A, +CC[_], +C] extends Any with IterableOnce[A] with Iterable
     (fromSpecific(first), fromSpecific(second))
   }
 
-  /** Splits this $coll into two at a given position.
-    *  Note: `c splitAt n` is equivalent to (but possibly more efficient than)
-    *         `(c take n, c drop n)`.
-    *  $orderDependent
-    *
-    *  @param n the position at which to split.
-    *  @return  a pair of ${coll}s consisting of the first `n`
-    *           elements of this $coll, and the other elements.
-    */
-  def splitAt(n: Int): (C, C) = (take(n), drop(n))
+  override def splitAt(n: Int): (C, C) = (take(n), drop(n))
```

Comment:
Maybe it’s better to move this override to `collection.IndexedSeqOps` instead, what do you think? The implementation inherited from `IterableOnceOps` might be better for other subtypes of iterables (`LinearSeq`, `Set` and `Map`) since it performs only one traversal.

Reply:
I don't think that would make any difference.. It would still dynamically dispatch to `span`, which is overridden to `(takeWhile(p), dropWhile(p))` - you end up with the same multiple traversal.